style.css

```diff
@@ -0,0 +1,37 @@
+#main {
+    align-content: center;
+    background-color: #eeeee4;
+}
+
+#title {
+    padding-top: 50px;
+    text-align: center;
+    font-size: 50px;
+    color: #873e23;
+    font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+}
+
+#image {
+    position: relative;
+    margin-left: 200px;
+}
+
+#img-caption  {
+    text-align: center;
+    font-style: italic;
+}
+
+#tribute-info h3 {
```

Comment:
Please use the proper indentations(2 spaces) in your css file. Use of improper indentations will not allow your linters to pass.